src/routes/Logout/components/LogoutPage/LogoutPage.js

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect } from 'react';
+import { useFirebaseApp } from 'reactfire';
+import { useHistory } from 'react-router-dom';
+import {
+  makeStyles,
+  Container,
+  Typography,
+  Paper,
+  LinearProgress,
+} from '@material-ui/core';
+import { Helmet } from 'react-helmet';
+import styles from './LogoutPage.styles';
+
+const useStyles = makeStyles(styles);
+
+function LogoutPage() {
+  const classes = useStyles();
+  const firebase = useFirebaseApp();
+  const history = useHistory();
+
+  useEffect(() => {
```

Comment:
If we don't make this a page, then we shouldn't need any of the useEffect stuff, we can just have the user logout from the dropdown

Reply:
In some places like the new user profile page, I felt it was simpler to redirect to this page than worrying about firebase auth, related possible render cycles and then redirecting.  Not a big deal either way, but this was simpler for me (although, a little more work).